examples/js/loaders/SVGLoader.js

```diff
@@ -361,6 +374,10 @@ THREE.SVGLoader.prototype = {
 					case 'Z':
 					case 'z':
 						path.currentPath.autoClose = true;
+						// Reset point to beginning of Path
+						point.x = path.currentPath.curves[ 0 ].v0.x;
```

Comment:
Selecting the first curve in the array is fine. But accessing `v0` is critical because not all curves have this property e.g `LineCurve` or `EllipseCurve`. Since there is not API for retrieving the first point of a path/curve, I guess you have to handle different cases.